docs/components/toaster.md

```diff
@@ -71,38 +71,38 @@ interface Toast {
 
 ### title
 
-Notification title. When passing an object to `$toaster.open()` method, the `title` property is required. When passing only a string, the string will be rendered as the title.
+Notification title.
 
-<div class="horizontal-container">
-  <KButton @click="$toaster.open({ title: 'Long Title Gets Truncated With Ellipsis' })">Open Toaster</KButton>
-  <KButton @click="$toaster.open('String Will Become A Title')" appearance="secondary">Open Toaster</KButton>
-</div>
+<KButton @click="$toaster.open({ title: 'Notification Title' })">Open Toaster</KButton>
 
 ```html
-<KButton @click="$toaster.open({ title: 'Long Title Gets Truncated With Ellipsis' })">Open Toaster</KButton>
-<KButton @click="$toaster.open('String Will Become A Title')" appearance="secondary">Open Toaster</KButton>
+<KButton @click="$toaster.open({ title: 'Notification Title' })">Open Toaster</KButton>
 ```
 
 ### message
 
-The message prop allows for displaying longer strings of text to the user. This prop is good for more detailed messages, or displaying IDs, etc.
+The message string that allows for displaying longer strings of text to the user. This prop is good for more detailed messages.
 
-Because a long title gets truncated, it is only good for short notifications. When a longer notification needs to be shown to the user, the `message` property is ideal.
+Alternatively, if you provide a string as the only argument, it will be treated as message.
```

Comment:
If you pass a string as the only argument to what?

Reply:
It's clearer in context - we are talking about arguments passed to `ToastManager.open()` method in this section.

![Screenshot 2024-02-21 at 9 02 37 AM](https://github.com/Kong/kongponents/assets/36751160/bd852e41-18f1-4c79-9fc0-a0bdbe55aab7)

Reply:
Since it's further down the page, how about something like this?

```md
Alternatively, if you provide a string as the only argument to the `open()` method, it will be treated as the message.
```

Reply:
Updated